mathesar_ui/src/sections/table-view/Header.svelte

```diff
@@ -149,11 +180,43 @@
       </Dropdown>
     </div>
   {/each}
+
   <div class="cell" style="width:{70 + DEFAULT_ROW_RIGHT_PADDING}px;left:{
-      columnPosition.get('__row').width + paddingLeft
-    }px;">
-    <div class="add">
-      +
-    </div>
+    columnPosition.get('__row').width + paddingLeft
+  }px;">
+    <Dropdown closeOnInnerClick={false}
+              contentClass="content"
```

Comment:
Default value of `closeOnInnerClick` is `false`, and `content` is a class that is already assigned to the dropdown's content by default. 

These values need not be explicitly mentioned when using a component.

Reply:
I have removed the unneeded props, thank you!

Reply:
https://github.com/pegnott/mathesar/blob/754414d202a29033bd437f2d845269d5885a5831/mathesar_ui/src/sections/table-view/Header.svelte#L223